en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -10,6 +10,12 @@ include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
 
 * You can now add custom logic, such as `and`, `or`, or `not`, in conditional statements when building an action and in If/Else statements.
 
+* In RPA Connector, you can now test an RPA process using sample data without actually invoking the RPA process.
+
+* In RPA Connector, if you are connected to your RPA organization but no processes appear in the dropdown for the *RPA Process* field, you can now connect directly to your RPA Organization to see your processes by clicking the *Create an invokable run config from MuleSoft RPA* button.
```

Comment:
```suggestion
* In RPA Connector, you can now connect directly to your RPA Organization to see your processes by clicking the *Create an invokable run config from MuleSoft RPA* button when you are connected to your RPA organization and no processes appear in the dropdown for the *RPA Process* field.
```

Reply:
I feel like that sentence is way too long but I couldn't think of a way to shorten it either. I do feel like the order should be reversed though to tell them what they can now do first.